release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,62 @@
-
-|Auth backend|`@backstage/plugin-auth-backend`|0.19.4|Security
-
-|Auth node|`@backstage/plugin-auth-node`|0.4.0|Security
-
-|Catalog frontend|`@backstage/plugin-catalog`|1.14.0|Continuous deployment
+=== Configuration change in the GitHub Org catalog provider  
 
-|Catalog backend|`@backstage/plugin-catalog-backend`|1.14.0|Continuous deployment
+The GitHub Org catalog module for the new backend system switched from an obsolete implementation to a new one, that is, `GithubMultiOrgProvider`. This new implementation brings in changes to the default plugin configuration settings.
 
-|Catalog common|`@backstage/plugin-catalog-common`|1.0.17|Continuous deployment
+=== New Scaffolder Identity client
 
-|Catalog Graph|`@backstage/plugin-catalog-graph`|0.2.37|Continuous deployment
+Identity client functioning differs slightly between the one provided by the new backend system Scaffolder plugin, and the one used in the previous release:
 
-|Catalog Import|`@backstage/plugin-catalog-import`|0.10.1|Continuous deployment
+* The previous release uses the `DefaultIdentityClient` function without any specific option
+* New backend system Scaffolder does not use the default identity service, but still uses the deprecated scaffolder-dedicated `buildDefaultIdentityClient` function, which introduces a slight functioning change in the identity client. 
 
-|Catalog React|`@backstage/plugin-catalog-react`|1.8.5|Continuous deployment
+=== Enable all GitLab actions
+{product} {product-version} enables all GitLab actions by default.
 
-|Events backend|`@backstage/plugin-events-backend`|0.2.15|Continuous deployment
+=== Events support redesign in progress for the new backend system
+The `Events` plugin is not completely supported by the new backend system, and thus its usefulness is greatly reduced, if not completely broken.
+A complete redesign of the `Events` plugin support for the new backend system is in progress, but was still not available to integrate it to this release of {product}. This limitation temporarily prevents third parties from fully using events in plugins or modules they want to provide as dynamic plugins for {product}.
 
-|Events node|`@backstage/plugin-events-node`|0.2.15|Continuous deployment
+=== RBAC Plugin
+In the new backend system, automatically getting the IDs of all dynamically installed plugins is not possible. This means that permissions of dynamic plugins supporting RBAC will not be automatically displayed in the RBAC UI.
+As a workaround, you can manually add a list of dynamic plugin IDs supporting RBAC in the {product-short} application configuration file.
 
-|Home|`@backstage/plugin-home`|0.5.9|Continuous deployment
 
-|Kubernetes backend|`@backstage/plugin-kubernetes-backend`|0.13.0|Container management
+== Support for Elastic Kubernetes Services (EKS)
 
-|Org|`@backstage/plugin-org`|0.6.15|Identity and access management
+You can now install and use {product} on an EKS cluster. 
```

Comment:
```suggestion
You can now install and use the {product} on an EKS cluster. 
```

Reply:
We do not need an article with the product name, specifically if we have not used any entity type after its actual name.